### DISCUSSION TITLE
How to discuss potential security vulnerabilities?

### DISCUSSION THREAD

**Discussion by Brutus5000** [yahoo/elide#2168] [General]
Hey guys, I might found a small low risk security vulnerability.
Back in "the days" I was just bothering @aklish on Slack instead of raising an issue, but I guess you are not using that anymore.
So what is the process now?

**Comment by aklish** [comment]
We will be adding another chat option here soon (probably in late July or August).  In the meantime, you can ping me on either slack or spectrum.
